### ISSUE TITLE
DropdownButton Accessibility Issues

### ISSUE THREAD

**Issue by jrios** [WTW-IM/es-components#205]
There are some accessibility issues that need to be addressed with the DropdownButton component:

1. Focus should be trapped within buttons until it is collapsed
2. Screen readers should be able to reliably state when the dropdown is expanded vs collapsed
